janus_core/calculations/md.py

```diff
@@ -463,6 +478,48 @@ def __init__(
         if not self.struct.calc:
             raise ValueError("Please attach a calculator to `struct`.")
 
+        # Wrap calculator with Plumed if input is provided
+        if self.plumed_input:
+            try:
+                from ase.calculators.plumed import Plumed
+            except ImportError:
+                raise ImportError(
+                    "ASE Plumed calculator not found. Install janus-core with "
+                    "'plumed' extra: pip install janus-core[plumed]"
+                )
```

Comment:
I'm not sure this ever actually generates an `ImportError`, so we may need to check `plumed` explicitly:

```
>>> from ase.calculators.plumed import Plumed
>>>
>>> from plumed import Plumed as pl
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'plumed'
```

I probably wouldn't specify exactly how to install plumed either.